.github/workflows/build_pr_documentation.yml

```diff
@@ -17,11 +17,11 @@ on:
       # supply --not_python_module for HF Course
       additional_args:
         type: string
-        default: ''
       languages:
       # supply space-separated language codes
         type: string
-        default: ''
+      hub_base_path:
```

Comment:
I chose to support just the base URL as I couldn't quite figure out how to propagate the other input parameters at this part of the workflow (the GitHub [docs](https://docs.github.com/en/actions/learn-github-actions/finding-and-customizing-actions#using-inputs-and-outputs-with-an-action) are a bit bare)